<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>685</width>
    <height>455</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <layout class="QVBoxLayout" name="verticalLayout_9">
    <item>
     <layout class="QVBoxLayout" name="verticalLayout_8">
      <item>
       <layout class="QVBoxLayout" name="verticalLayout">
        <item>
         <layout class="QHBoxLayout" name="horizontalLayout_3">
          <item>
           <widget class="QGroupBox" name="groupBox2">
            <property name="title">
             <string>Поиск</string>
            </property>
            <layout class="QHBoxLayout" name="horizontalLayout_2">
             <item>
              <widget class="QLineEdit" name="searchEdit"/>
             </item>
            </layout>
           </widget>
          </item>
          <item>
           <widget class="QGroupBox" name="groupBox">
            <property name="title">
             <string>Дата</string>
            </property>
            <layout class="QHBoxLayout" name="horizontalLayout">
             <item>
              <widget class="QCheckBox" name="anniverBtn">
               <property name="text">
                <string>Юбилейные</string>
               </property>
              </widget>
             </item>
             <item>
              <widget class="QLabel" name="label">
               <property name="text">
                <string>День</string>
               </property>
              </widget>
             </item>
             <item>
              <widget class="QComboBox" name="dayBox">
               <property name="enabled">
                <bool>true</bool>
               </property>
               <property name="editable">
                <bool>false</bool>
               </property>
               <item>
                <property name="text">
                 <string>любой</string>
                </property>
               </item>
               <item>
                <property name="text">
                 <string>1</string>
                </property>
               </item>
               <item>
                <property name="text">
                 <string>2</string>
                </property>
               </item>
               <item>
                <property name="text">
                 <string>3</string>
                </property>
               </item>
               <item>
                <property name="text">
                 <string>4</string>
                </property>
               </item>
               <item>
                <property name="text">
                 <string>5</string>
                </property>
               </item>
               <item>
                <property name="text">
                 <string>6</string>
                </property>
               </item>
               <item>
                <property name="text">
                 <string>7</string>
                </property>
               </item>
               <item>
                <property name="text">
                 <string>8</string>
                </property>
               </item>
               <item>
                <property name="text">
                 <string>9</string>
                </property>
               </item>
               <item>
                <property name="text">
                 <string>10</string>
                </property>
               </item>
               <item>
                <property name="text">
                 <string>11</string>
                </property>
               </item>
               <item>
                <property name="text">
                 <string>12</string>
                </property>
               </item>
               <item>
                <property name="text">
                 <string>13</string>
                </property>
               </item>
               <item>
                <property name="text">
                 <string>14</string>
                </property>
               </item>
               <item>
                <property name="text">
                 <string>15</string>
                </property>
               </item>
               <item>
                <property name="text">
                 <string>16</string>
                </property>
               </item>
               <item>
                <property name="text">
                 <string>17</string>
                </property>
               </item>
               <item>
                <property name="text">
                 <string>18</string>
                </property>
               </item>
               <item>
                <property name="text">
                 <string>19</string>
                </property>
               </item>
               <item>
                <property name="text">
                 <string>20</string>
                </property>
               </item>
               <item>
                <property name="text">
                 <string>21</string>
                </property>
               </item>
               <item>
                <property name="text">
                 <string>22</string>
                </property>
               </item>
               <item>
                <property name="text">
                 <string>23</string>
                </property>
               </item>
               <item>
                <property name="text">
                 <string>24</string>
                </property>
               </item>
               <item>
                <property name="text">
                 <string>25</string>
                </property>
               </item>
               <item>
                <property name="text">
                 <string>26</string>
                </property>
               </item>
               <item>
                <property name="text">
                 <string>27</string>
                </property>
               </item>
               <item>
                <property name="text">
                 <string>28</string>
                </property>
               </item>
               <item>
                <property name="text">
                 <string>29</string>
                </property>
               </item>
               <item>
                <property name="text">
                 <string>30</string>
                </property>
               </item>
               <item>
                <property name="text">
                 <string>31</string>
                </property>
               </item>
              </widget>
             </item>
             <item>
              <widget class="QLabel" name="label_2">
               <property name="text">
                <string>Месяц</string>
               </property>
              </widget>
             </item>
             <item>
              <widget class="QComboBox" name="monthBox">
               <property name="editable">
                <bool>false</bool>
               </property>
               <item>
                <property name="text">
                 <string>любой</string>
                </property>
               </item>
               <item>
                <property name="text">
                 <string>Январь</string>
                </property>
               </item>
               <item>
                <property name="text">
                 <string>Февраль</string>
                </property>
               </item>
               <item>
                <property name="text">
                 <string>Март</string>
                </property>
               </item>
               <item>
                <property name="text">
                 <string>Апрель</string>
                </property>
               </item>
               <item>
                <property name="text">
                 <string>Май</string>
                </property>
               </item>
               <item>
                <property name="text">
                 <string>Июнь</string>
                </property>
               </item>
               <item>
                <property name="text">
                 <string>Июль</string>
                </property>
               </item>
               <item>
                <property name="text">
                 <string>Август</string>
                </property>
               </item>
               <item>
                <property name="text">
                 <string>Сентябрь</string>
                </property>
               </item>
               <item>
                <property name="text">
                 <string>Октябрь</string>
                </property>
               </item>
               <item>
                <property name="text">
                 <string>Ноябрь</string>
                </property>
               </item>
               <item>
                <property name="text">
                 <string>Декабрь</string>
                </property>
               </item>
              </widget>
             </item>
             <item>
              <widget class="QLabel" name="label_3">
               <property name="text">
                <string>Год</string>
               </property>
              </widget>
             </item>
             <item>
              <widget class="QLineEdit" name="yearEdit">
               <property name="sizePolicy">
                <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
                 <horstretch>0</horstretch>
                 <verstretch>0</verstretch>
                </sizepolicy>
               </property>
               <property name="maximumSize">
                <size>
                 <width>60</width>
                 <height>16777215</height>
                </size>
               </property>
              </widget>
             </item>
            </layout>
           </widget>
          </item>
         </layout>
        </item>
        <item>
         <layout class="QHBoxLayout" name="horizontalLayout_6">
          <item>
           <widget class="QListWidget" name="listWidget">
            <property name="sizePolicy">
             <sizepolicy hsizetype="Fixed" vsizetype="Expanding">
              <horstretch>0</horstretch>
              <verstretch>0</verstretch>
             </sizepolicy>
            </property>
            <property name="maximumSize">
             <size>
              <width>150</width>
              <height>16777215</height>
             </size>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QTableWidget" name="tableWidget">
            <property name="maximumSize">
             <size>
              <width>16777215</width>
              <height>16777215</height>
             </size>
            </property>
            <property name="contextMenuPolicy">
             <enum>Qt::CustomContextMenu</enum>
            </property>
            <property name="editTriggers">
             <set>QAbstractItemView::NoEditTriggers</set>
            </property>
            <property name="alternatingRowColors">
             <bool>false</bool>
            </property>
            <property name="selectionMode">
             <enum>QAbstractItemView::SingleSelection</enum>
            </property>
            <property name="selectionBehavior">
             <enum>QAbstractItemView::SelectRows</enum>
            </property>
            <property name="showGrid">
             <bool>true</bool>
            </property>
            <property name="columnCount">
             <number>5</number>
            </property>
            <attribute name="horizontalHeaderCascadingSectionResizes">
             <bool>false</bool>
            </attribute>
            <attribute name="horizontalHeaderDefaultSectionSize">
             <number>100</number>
            </attribute>
            <attribute name="horizontalHeaderStretchLastSection">
             <bool>false</bool>
            </attribute>
            <column>
             <property name="text">
              <string>Дата</string>
             </property>
            </column>
            <column>
             <property name="text">
              <string>Событие</string>
             </property>
            </column>
            <column>
             <property name="text">
              <string>Фото</string>
             </property>
            </column>
            <column>
             <property name="text">
              <string>Место</string>
             </property>
            </column>
            <column>
             <property name="text">
              <string>Источник</string>
             </property>
            </column>
           </widget>
          </item>
         </layout>
        </item>
       </layout>
      </item>
      <item>
       <widget class="QGroupBox" name="detailBox">
        <property name="sizePolicy">
         <sizepolicy hsizetype="Preferred" vsizetype="Fixed">
          <horstretch>0</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
        <property name="maximumSize">
         <size>
          <width>16777215</width>
          <height>200</height>
         </size>
        </property>
        <property name="title">
         <string>Событие</string>
        </property>
        <layout class="QHBoxLayout" name="horizontalLayout_5">
         <item>
          <layout class="QVBoxLayout" name="verticalLayout_7">
           <item>
            <layout class="QHBoxLayout" name="horizontalLayout_4">
             <item>
              <layout class="QVBoxLayout" name="verticalLayout_2">
               <item>
                <widget class="QLabel" name="label_4">
                 <property name="text">
                  <string>Дата</string>
                 </property>
                </widget>
               </item>
               <item>
                <widget class="QLineEdit" name="dateEdit">
                 <property name="readOnly">
                  <bool>true</bool>
                 </property>
                </widget>
               </item>
              </layout>
             </item>
             <item>
              <layout class="QVBoxLayout" name="verticalLayout_3">
               <item>
                <widget class="QLabel" name="label_5">
                 <property name="text">
                  <string>Тематика</string>
                 </property>
                </widget>
               </item>
               <item>
                <widget class="QLineEdit" name="themeEdit">
                 <property name="readOnly">
                  <bool>true</bool>
                 </property>
                </widget>
               </item>
              </layout>
             </item>
            </layout>
           </item>
           <item>
            <layout class="QVBoxLayout" name="verticalLayout_4">
             <item>
              <widget class="QLabel" name="label_6">
               <property name="text">
                <string>Место</string>
               </property>
              </widget>
             </item>
             <item>
              <widget class="QLineEdit" name="placeEdit">
               <property name="readOnly">
                <bool>true</bool>
               </property>
              </widget>
             </item>
            </layout>
           </item>
           <item>
            <layout class="QVBoxLayout" name="verticalLayout_5">
             <item>
              <widget class="QLabel" name="label_8">
               <property name="text">
                <string>Дополнительно</string>
               </property>
              </widget>
             </item>
             <item>
              <widget class="QLineEdit" name="extraEdit">
               <property name="readOnly">
                <bool>true</bool>
               </property>
              </widget>
             </item>
            </layout>
           </item>
          </layout>
         </item>
         <item>
          <layout class="QVBoxLayout" name="verticalLayout_6">
           <item>
            <widget class="QLabel" name="label_7">
             <property name="text">
              <string>Подробное описание</string>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QPlainTextEdit" name="fullEdit">
             <property name="readOnly">
              <bool>true</bool>
             </property>
            </widget>
           </item>
          </layout>
         </item>
        </layout>
       </widget>
      </item>
     </layout>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>685</width>
     <height>27</height>
    </rect>
   </property>
   <property name="sizePolicy">
    <sizepolicy hsizetype="MinimumExpanding" vsizetype="Minimum">
     <horstretch>0</horstretch>
     <verstretch>0</verstretch>
    </sizepolicy>
   </property>
   <widget class="QMenu" name="menu">
    <property name="title">
     <string>Даты</string>
    </property>
    <addaction name="newEventAction"/>
    <addaction name="themeAction"/>
    <addaction name="placeAction"/>
   </widget>
   <widget class="QMenu" name="menu_2">
    <property name="title">
     <string>Вид</string>
    </property>
    <addaction name="increaseFont"/>
    <addaction name="decreaseFont"/>
    <addaction name="action_10"/>
    <addaction name="detailAction"/>
   </widget>
   <widget class="QMenu" name="menu_5">
    <property name="title">
     <string>Файл</string>
    </property>
    <addaction name="action_2"/>
    <addaction name="exportAction"/>
    <addaction name="settingsAction"/>
    <addaction name="helpAction"/>
    <addaction name="exitAction"/>
   </widget>
   <addaction name="menu_5"/>
   <addaction name="menu"/>
   <addaction name="menu_2"/>
  </widget>
  <action name="newEventAction">
   <property name="text">
    <string>Новое событие</string>
   </property>
  </action>
  <action name="action_2">
   <property name="text">
    <string>Импорт из файла</string>
   </property>
  </action>
  <action name="exportAction">
   <property name="text">
    <string>Экспорт в файл</string>
   </property>
  </action>
  <action name="themeAction">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="text">
    <string>Тематика</string>
   </property>
  </action>
  <action name="placeAction">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="text">
    <string>Место</string>
   </property>
  </action>
  <action name="exitAction">
   <property name="text">
    <string>Выход</string>
   </property>
  </action>
  <action name="increaseFont">
   <property name="text">
    <string>Шрифт крупнее</string>
   </property>
  </action>
  <action name="decreaseFont">
   <property name="text">
    <string>Шрифт меньше</string>
   </property>
  </action>
  <action name="anniverAction">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="text">
    <string>Юбилейные</string>
   </property>
  </action>
  <action name="action_10">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="text">
    <string>С фотографиями</string>
   </property>
  </action>
  <action name="detailAction">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="text">
    <string>Показать подробнее</string>
   </property>
  </action>
  <action name="settingsAction">
   <property name="text">
    <string>Настройки</string>
   </property>
  </action>
  <action name="helpAction">
   <property name="text">
    <string>Справка</string>
   </property>
  </action>
  <action name="cardAction">
   <property name="text">
    <string>Карточка</string>
   </property>
  </action>
  <action name="fullList">
   <property name="text">
    <string>Показать полный список</string>
   </property>
  </action>
  <action name="removeEvent">
   <property name="text">
    <string>Удалить</string>
   </property>
  </action>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections/>
</ui>
